src/components/tree/daikin-tree.ts

```diff
@@ -51,63 +51,58 @@ export class DaikinTree extends LitElement {
   /**
    * Whether or not to enable tree selection.
    * When enabled, tree sections and items can be selected by click, and the `selected` property of the `daikin-tree` and its descendants will be automatically controlled.
-   * Even if this is disabled, you can still set the `selected` property of sections and items yourself.
+   * Even if this is disabled, you can still set the `selected` property yourself.
    */
   @property({ type: Boolean, reflect: true })
   selectable: boolean = false;
 
   /**
    * The value of the currently selected tree section or tree item.
-   * Only used if `selectable` is `true`.
+   * Even if `selectable` is `false`, you can still set this property yourself.
    */
-  @property({ type: String, reflect: true })
-  selected: string | null = null;
+  @property({ type: Array, attribute: false })
+  selectedItems: string[] = [];
 
   @queryAssignedElements({ selector: "daikin-tree-section,daikin-tree-item" })
   private readonly _children!: readonly (DaikinTreeSection | DaikinTreeItem)[];
 
-  @queryAssignedElements({ selector: "daikin-tree-section" })
-  private readonly _sections!: readonly DaikinTreeSection[];
-
-  private _updateChildrenLevel(): void {
-    this._children.forEach((child) => (child.level = 0));
-  }
-
-  private _updateSections(): void {
-    this._sections.forEach((section) => {
-      section.selectable = this.selectable;
-    });
+  private _handleMouseDown(event: MouseEvent) {
+    if (event.detail === 2) {
+      // Prevent text selection on double click.
```

Comment:
It seems that we should also handle the tree with three or more clicks. (Though I don't know about the dropdown.)

```suggestion
    if (event.detail >= 2) {
      // Prevent text selection on double click, triple-click, and so on.
```